projects/aws/upgrader/Makefile

```diff
@@ -12,15 +13,25 @@ EXCLUDE_FROM_UPGRADE_BUILDSPEC=true
 # for the staging buildspec generation
 BUILDSPEC_DEPENDS_ON_OVERRIDE=containerd_containerd_linux_amd64 containerd_containerd_linux_arm64 kubernetes_sigs_cri_tools
 
+BINARY_TARGET_FILES=upgrader
+SOURCE_PATTERNS=. 
+GO_MOD_PATHS=..
+
+# force binaries to go to non-release branch bin folder
+BINARIES_ARE_RELEASE_BRANCHED=false
+GIT_CHECKOUT_TARGET=main.go
+FAKE_GIT_REPO_TARGET=.git
+REPO_NO_CLONE=true
+HAS_LICENSES=false
+SKIP_CHECKSUM_VALIDATION=true
```

Comment:
Let's keep license generation and checksum validation for this project.

Reply:
yeah remove those two lines, 25 and 26

Reply:
Looks like we already had the `HAS_LICENSES=false` value set in our previous make. did something change between then and now to not have it .?

Reply:
yes its a go bin now so its deps and therefore licenses and attribtion.